### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by darshandamre**
add home page of e-commerce site 
Preview [https://e-com-git-home-page-darshandamre.vercel.app](https://e-com-git-home-page-darshandamre.vercel.app/)

### CODE REVIEW

File: global.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+.logo {
+  font-family: "Orbitron", sans-serif;
+  font-size: 2rem;
+  line-height: 2.5rem;
+}
+
+.container {
+  width: 90%;
+  max-width: 1920px;
```
Comment (akash-1318) [inline global.css:84]: use rem instead of pixel

**Review by akash-1318 — COMMENTED** [review]
Overall code looks clean to me just one minor change i have added.

**Review by arjunbharti — COMMENTED** [review]
Everything looks good. The UI is really nice. Good work!

File: global.css (hunk 1 above)
Comment (darshandamre) [reply global.css:84]: i've used px here so that this value won't change even after changing the root font-size in media-queries

**Pull request merged**